Readme.md

```diff
@@ -10,7 +10,7 @@ Do you want to quickly try out this project and see it in action? Check it out i
 
 ## Prerequisites
 
-1. This project relies on [Lombiq Node.js Extensions](https://github.com/Lombiq/NodeJs-Extensions/), please see its pre-requisites [here](https://github.com/Lombiq/NodeJs-Extensions/#pre-requisites).
+1. This project relies on [Lombiq Node.js Extensions](https://github.com/Lombiq/NodeJs-Extensions/), please see its prerequisites [here](https://github.com/Lombiq/NodeJs-Extensions/#prerequisites).
```

Comment:
Why is this change necessary? Can be avoided

Reply:
What do you mean? Spelling should be  consistent across the document (and across our repos) so this is a good change.